package.json

```diff
@@ -28,9 +28,6 @@
     "email": "ismaelga@gmail.com"
   },
   "main": "dist/react-json-editor.js",
-  "peerDependencies": {
-    "react": ">=0.12.2 <1.0.0"
-  },
```

Comment:
@marksantcroos I ended up dropping my dependency on this project. Though, last I checked, the change @ismaelga suggested worked fine, so you can probably fork the repo, make the change and use the fork. Or submit a PR and hope this repo is still being maintained.